BunqSdk.Tests/Model/Generated/Endpoint/RequestInquiryTest.cs

```diff
@@ -16,21 +16,21 @@ public class RequestInquiryTest : BunqSdkTestBase
         /// <summary>
         /// Config values.
         /// </summary>
-        private const string AMOUNT_EUR = "0.01";
-        private const string FIELD_CURRENCY = "EUR";
-        private const string FIELD_PAYMENT_DESCRIPTION = "C# test Payment";
-        private const string FIELD_STATUS = "ACCEPTED";
-        private const int INDEX_FIRST = 0;
+        private const string RequestInquiryAmountEur = "0.01";
```

Comment:
I'd change naming to something like ValueAmountEur (indicating that it is a value to be used in the test). RequestInquiryStatus looks a lot like the actual status objects now imo (@OGKevin 👀 )

Reply:
@sandervdo Argh you have a point 😭 stupid naming convention. @antao lets group these values by `Value` then 😒

Reply:
Will change this group with the keyword 'Value' then. 👍 @OGKevin @sandervdo is there a coding style guide available? That would be 👌

Reply:
Hmm No there is no coding style publicly available. ATM we have a mixture of the Microsoft Conde stye and or own. E.g. the constants grouping and no plurals. Might be a good idea to write a wiki about the code style 👍

Reply:
Okay, last question. Shall we also change the constants to contain 'Value' grouping in the test sample classes? Thanks guys!

Reply:
Yes, lets be consistent 👍 

And thank you 🤗

Reply:
Cool, last remark, Imho I believe we should use Value group in AmountEur or CurrencyEur in the case of this constants. And also group with value Field for others used in Field constants. What do you think ?

Reply:
Example: ![image](https://user-images.githubusercontent.com/429267/34412365-2578ab86-ebdd-11e7-9dd6-338e6227de9d.png)

Reply:
All `Status` and `description` are not fields tho, those are also values 😁 .

A field typically looks like this:
```csharp
        public const string FieldStatus = "status";
```
Instead of the actual value that the status field will hold, in this case `ACCEPTED`.